tests/api/route-services/__snapshots__/pipeline-response.test.js.snap

```diff
@@ -12,7 +12,7 @@ exports[`Test Pipeline Response Service functions propely with correct input (cu
         },
         "Key": Object {
           "experimentId": Object {
-            "S": "1234",
+            "S": undefined,
```

Comment:
According to the test, the value of `experimentId` is passed in the test, while it was previously passed. Are you sure this is the expected behaviour?

Reply:
You're right, I forgot to adapt the fake message to the new structure received.